sycl/test-e2e/Matrix/XMX8/joint_matrix_out_bounds.cpp

```diff
@@ -0,0 +1,22 @@
+//==-------- joint_matrix_out_bounds.cpp - DPC++ joint_matrix--------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: matrix-xmx8
+
+// RUN: %{build} -o %t.out -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4
+// RUN: %{run} %t.out
+
+#include <iostream>
```

Comment:
Lines 13 to 18 can be moved to .hpp file from both .cpp files to reduce code duplication.